cfgov/jinja2/v1/activity-log/index.html

```diff
@@ -3,7 +3,7 @@
 {% block content_main %}
 
     <div class="block block__flush-top">
-        <h1 data-qa-hook="main-title">Activity Log</h1>
+        <h1 data-qa-hook="main-title">Recent updates</h1>
```

Comment:
Was the change from capital `L` to lowercase `u` intentional?

Reply:
Yes, it should be sentence case, see https://[GHE]/CFGOV/platform/issues/1112#issuecomment-126298 for background.